app/controllers/spree/admin/taxjar_settings_controller.rb

```diff
@@ -58,6 +58,36 @@ def api_sync
         end
         redirect_back(fallback_location: spree.admin_taxjar_settings_path)
       end
+
+      def cache_tax_categories
```

Comment:
Would `cached_tax_categories` be a better name here, since we rely on this returning the tax categories?

Reply:
Yeah, good suggestion.